src/content/configuration/entry-context.md

```diff
@@ -75,4 +79,6 @@ module.exports = {
 };
 ```
 
+> For example: you can use dynamic entries to get the actual entries from an external source (remote server, file system content or database)
```

Comment:
If we are going to provide an example, don't put it into a note block:
```suggestion
For example: you can use dynamic entries to get the actual entries from an external source (remote server, file system content or database)
```

Also when we provide any example context, code block should follow. Please add abstract example in format of:

__webpack.config.js__

```javascript
// webpack config with abstract example corresponding to the given context
```

Reply:
fixed, thanks